design-toolkit/components/src/index.css

```diff
@@ -234,6 +234,14 @@
   --icon-color: --value(--color- *);
 }
 
+@custom-variant data-selected (&[data-selected~="true"]);
+@custom-variant data-active (&[data-active~="true"]);
+@custom-variant data-pinned (&:is([data-pinned~="top"], [data-pinned~="bottom"]));
```

Comment:
Let's remove the data-

We should make it clear whether this is a boolean and/or enum type of selector.

As an enum selector, you would have value specific selectors:
`@custom-variant pinned-top (&[data-pinned="top"])`

As a boolean selector, you would have:
`@custom-variant pinned (&[data-pinned])`

Also, thank you for teaching me the shorthand and not requiring the use of `@slot`. I can go back and simplify the other variant selectors